apps/website/src/content/docs/table.mdx

```diff
@@ -8,14 +8,360 @@ thumbnail: Table
 
 <Placeholder componentPageName='table--basic' />
```

Comment:
The placeholder has served its purpose, let's remove it.

If there are any features not documented yet, we can link to specific stories at the bottom of the page under a "More examples" heading.